perfkitbenchmarker/data/cassandra/cassandra-env.sh.j2

```diff
@@ -6,7 +6,7 @@
 # "License"); you may not use this file except in compliance
```

Comment:
cassandra-env.sh adds one line under `# Added for PerfKitBenchmarker`.
cassandra.yaml changes are noted with comments.